src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ReferenceNumberRepository.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.cmc.claimstore.repositories;
+
+import org.skife.jdbi.v2.sqlobject.SqlQuery;
+
+public interface ReferenceNumberRepository {
+
+    @SqlQuery("SELECT next_reference_number()")
+    String getReferenceNoForCitizen();
```

Comment:
Can you please change `No` to `Number`? I found it slightly confusing at a first glance.